examples/advanced/create-react-app/style-dictionary/config.json

```diff
@@ -0,0 +1,22 @@
+{
```

Comment:
I like how you have a 'style-dictionary' directory with the config in there and the properties directory.

Reply:
👍